docs/how-to.asciidoc

```diff
@@ -4,20 +4,11 @@
 Learn how to perform common APM configuration and management tasks.
 
 * <<source-map-how-to>>
-* <<ilm-how-to>>
 * <<jaeger-integration>>
 * <<ingest-pipelines>>
-* <<manage-storage>>
-* <<apm-tune-elasticsearch>>
 
 include::./source-map-how-to.asciidoc[]
 
-include::./ilm-how-to.asciidoc[]
-
 include::./jaeger-integration.asciidoc[]
 
 include::./ingest-pipelines.asciidoc[]
-
-include::./manage-storage.asciidoc[]
-
-include::./apm-tune-elasticsearch.asciidoc[]
```

Comment:
Curious what y'all think about moving this content. This PR creates a top-level bucket called, "Manage Storage":
<img width="344" alt="Screen Shot 2021-11-08 at 12 36 15 PM" src="https://user-images.githubusercontent.com/5618806/140815034-859f9456-e6dc-475c-968f-28b8933becba.png">

Reply:
I like it. I'm not sure that "Processing and performance" belongs there though, as that's about more than just storage -- it also covers apm-server's processing overhead.